js/src/common/Application.js

```diff
@@ -367,8 +373,8 @@ export default class Application {
    * @param {string[]} [formattedError]
    * @private
    */
-  showDebug(error, formattedError) {
-    this.alerts.dismiss(this.requestError.alert);
+  showDebug(error) {
```

Comment:
`formattedError` should not have been removed. It is still used in the same method.

Reply:
Shoddy merge conflict resolution on my part, sorry about that.